### DISCUSSION TITLE
How can I set the default match detection?

### DISCUSSION THREAD

**Discussion by wrohdewald** [dani-garcia/vaultwarden#5265] [Q&A]
Since I have several subdomains (I believe you say "host"), I want to change the default match detection from standard to host, or I would have to manually edit quite a few entries (imported from keepassxc). 

The "help" link goes to bitwarden which allows to define the default in the user settings. I do not see this in vaultwarden.

https://bitwarden.com/help/uri-match-detection/

Is there some config file I can edit?

**Comment by BlackDex** [comment]
Not sure where you are looking then, since Vaultwarden only provides a slightly customized web-vault.

All should be the same and workable as described in that manual.

**Reply by stefan0xC** [reply]
Here's the [relevant part of the documentation](https://bitwarden.com/help/uri-match-detection/#default-match-detection):

> Bitwarden browser extensions and mobile apps allow you to select a **Default match detection** behavior from the options listed next by navigating to **Settings → Autofill → Default URI match detection**. You can override the globally-set detection on an item-by-item basis in all Bitwarden apps.
> 
> **Base domain** matching is the default option.

**Comment by arksega** [comment]
@wrohdewald I had this confusion one moth ago and I gave up on this unclear pointers.
The 'settings' the documentation is referring to is on the client site, not in vault warden itself:
![image](https://github.com/user-attachments/assets/b2fab344-22af-42e4-a867-d61f2cb9231c)

Then you can navigate to the button of autofill to change the default url match:
![image](https://github.com/user-attachments/assets/cbd30c70-c318-462e-88da-f483ca0c5506)

**Reply by arobs1** [reply]
thank you this was helpful to me
